### DISCUSSION TITLE
Should we add "Share on Social Media" button on all pages

### DISCUSSION THREAD

**Discussion by kapilraghuwanshi** [mdn/mdn-community#554] [Content]
I just think that I would like to add few buttons like share to Twitter, LinkedIn, Facebook, or Instagram, etc on every page here.
Basically, it will allow us to share specific topics or pages with everyone on social media.
What do you guys think of this share button?

**Comment by Ryuno-Ki** [comment]
Can we have the navigator.share API for this?
By listing a few blessed platforms, you're helping enshrining their market position. Thus, I'd like to have a way to decide based on the user preference what the share target might be.

Further reading: https://adactio.com/journal/17493

**Comment by pepelsbey** [comment]
Closing this discussion as it didn’t get much traction.

It might also become irrelevant. Feel free to start a new one if that’s not the case.
